joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -194,12 +196,15 @@ controller_interface::return_type JointTrajectoryController::update(
         traj_external_point_ptr_->set_point_before_trajectory_msg(
           time, state_current_, joints_angle_wraparound_);
       }
+      traj_external_point_ptr_->sample(
+        // Sample once at the beginning to establish the start time if none was given
+        time, interpolation_method_, state_desired_, start_segment_itr, end_segment_itr);
```

Comment:
```suggestion
      // Sample once at the beginning to establish the start time if none was given
      traj_external_point_ptr_->sample(
        time, interpolation_method_, state_desired_, start_segment_itr, end_segment_itr);
```

Just to avoid comment in the middle of the function arguments